### ISSUE TITLE
Touchscreen and keyboard accessibility

### ISSUE THREAD

**Issue by r-tae** [Gesture-and-Sign-Corpus-of-Australia/signbank#27]
Need to do a full pass of both touchscreen and keyboard accessibility. Failures will be captured here in a checklist
